.travis.yml

```diff
@@ -16,15 +16,18 @@ env:
   - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
   - DJANGO="django>=1.11,<1.12" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
   - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
-  - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
+  - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
+  - DJANGO="django>=2.0,<2.1" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
+  - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
+  - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
```

Comment:
remove this line because is redundant, it was also redundant before, so we'd better fix this according to the ["broken window" principle](https://pragprog.com/the-pragmatic-programmer/extracts/software-entropy)

Reply:
all of them?, ok